### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by pavloo**
addresses https://github.com/pavloo/config_py/issues/3

### CODE REVIEW

**Comment by pavloo** [comment]
@pzmudzinski A rendered version https://github.com/pavloo/config_py/blob/8ff711495119516a886d68a9bea4a65c2d01fa65/README.md

File: README.md [hunk 1]
```diff
@@ -1,2 +1,95 @@
 [![Build Status](https://travis-ci.org/pavloo/config_py.svg?branch=master)](https://travis-ci.org/pavloo/config_py)
-### TBD
+
+# Disclaimer: the library is under active development and not intended to be used right now
+
+## config_py
+*A Python package for managing application configurations based on [Convention Over Configuration](https://en.wikipedia.org/wiki/Convention_over_configuration) principle*.
+
```
Comment (pzmudzinski) [inline README.md]: I think we need some short explanation why you need this at all - e.g. 
Sample use cases: 
- You want use different URLs based on running in production and during development 
- You want to disable or enable some features in production but keep them on in development

File: README.md [hunk 2]
```diff
@@ -1,2 +1,95 @@
...
+### Installation
+```
+pip install config_py
+```
+
+### Basic usage
+This package provides a cli for scaffolding configuration setup. Assuming you have a Python package called `my_package` given the next directory structure:
+```
```
Comment (pzmudzinski) [inline README.md]: CLI

File: README.md [hunk 3]
```diff
@@ -1,2 +1,95 @@
...
+
+## config_py
+*A Python package for managing application configurations based on [Convention Over Configuration](https://en.wikipedia.org/wiki/Convention_over_configuration) principle*.
+
+### Installation
+```
+pip install config_py
+```
+
+### Basic usage
```
Comment (pavloo) [inline README.md:25]: need to add prerequisites (python3, pip)

**Pull request merged**